### PR TITLE
Linked all examples, headline capitalization, typography.

### PR DESCRIPTION
**Pull request by virtualritz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 
 ## Upcoming Features
- - ~~Support for isometric and hexagon rendering~~ done
- - ~~Built in animation support.~~ done see animation example
- - Texture array support
- - ~~Layers and add/remove tiles. (High Priority)~~ done
+ - [x] Support for isometric and hexagon rendering.
+ - [x] Built in animation support  – see [`animation` example](examples/animation.rs).
+ - [ ] Texture array support.
+ - [x] Layers and add/remove tiles..
```
Comment (StarArawn) [inline README.md:15]: Double period here?

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 ![hex](screenshots/hex.png)
 
-### How this works?
+### How Does This Work?
 Quite simple there is a tile per entity. Behind the scenes the tiles are split into chunks that each have their own mesh which is sent to the GPU in an optimal way.
 
-### Why use this over another bevy tile map plugin?
-Because each tile is an entity of its own editing tiles is super easy, and convenient. This allows you tag entities for updating and makes stuff like animation easier. Want to have a mining simulation where damage is applied to tiles? That's easy with this plugin:
+### Why Use This Instead of X?
+Because each tile is an entity of its own editing tiles is super easy and convenient. This allows you tag entities for updating and makes stuff like animation easier. Want to have a mining simulation where damage is applied to tiles? That’s easy with this plugin:
```
Comment (StarArawn) [inline README.md:26]: > This allows you tag entities for updating

I think this should say: "This allows you **to** tag entities for updating".

**Review by StarArawn — CHANGES_REQUESTED** [review]
Thank you so much for the changes here! I think it makes things a lot easier to read, but I did have a few minor suggestions.

**Comment by virtualritz** [comment]
Fixed (I hope).

**Review by StarArawn — APPROVED** [review]

**Pull request merged**